README.md

```diff
@@ -103,8 +124,8 @@ cargo run --release --example testbed
 cargo b --release --example node
 
 # Run example node
-export DUSK_WALLET_DIR="TODO"
-export DUSK_CONSENSUS_KEYS_PASS="TODO"
+export DUSK_WALLET_DIR="TBD"
+export DUSK_CONSENSUS_KEYS_PASS="TBD"
 
 USAGE:
     node --bootstrap <bootstrap>... --address <public_address>  --preloaded-num <preloaded-num> --provisioner-unique-id <prov-id>  --log-level <LOG>
```

Comment:
Maybe a short explanation of the arguments?

Reply:
This should be addressed by a separate issue which should add descriptions to the CLI params.

Reply:
Agreed